remap-keys.lisp

```diff
@@ -73,7 +76,9 @@
          (keys (cdr (assoc (print-key raw-key) keymap :test 'equal))))
     (when keys
       (dolist (key keys)
-        (send-fake-key window key))
+        (if (eq *remap-keys-enable* t)
```

Comment:
Could you move the conditional 'into' the arguments of `send-fake-key`. Also it is not necessary to `(eq t ...)`. Just use the value directly.

```
(send-fake-key window (if *remapped-keys-enabled-p*
                                           key
                                           raw-key)
```

Or bind the key value to use in an outer let, whichever you prefer

```
(let ((key (if *remapped-keys-enabled-p*
                   key
                   raw-key)))
  (send-fake-key window key)
```

Reply:
ok